packages/console/src/pages/ApiResources/index.tsx

```diff
@@ -130,9 +130,10 @@ const ApiResources = () => {
             }}
           />
         }
-        onClickRow={({ id }) => {
-          navigate(buildDetailsPathname(id));
-        }}
+        clickRowHandler={({ id }) =>
+          () => {
+            navigate(buildDetailsPathname(id));
+          }}
```

Comment:
What is the idea behind this update?

Reply:
The click behavior should be controlled at the row level because some rows may not allow clicking. So return a function here.